packages/nitro-protocol/src/index.ts

```diff
@@ -34,11 +34,10 @@ export {Channel, getChannelId} from './contract/channel';
 export {encodeConsensusData, decodeConsensusData, ConsensusData} from './contract/consensus-data';
 export {validTransition, ForceMoveAppContractInterface} from './contract/force-move-app';
 export {
-  AllocationItem,
   encodeAllocation,
   encodeOutcome,
   Outcome,
-  Allocation,
+  NitroAllocation as Allocation,
```

Comment:
I think contortions such as these are probably best avoided in this package which is at the very base of our stack.

Reply:
Ah I didn't notice this, will fix.